node_modules/oae-core/topnavigation/js/topnavigation.js

```diff
@@ -67,7 +67,8 @@ define(['jquery', 'oae.core', 'activityadapter'], function($, oae, ActivityAdapt
         var initLeftMenu = function() {
```

Comment:
@mrvisser : Does this mean that this is not a bug, but a feature we never really had ?